### PR TITLE
Create tictactoe.py final copy

### PR DESCRIPTION
**Pull request by dandre2**

### CODE REVIEW

File: DA/submissions/tictactoe.py final copy [hunk 1]
```diff
@@ -0,0 +1,124 @@
+def player1wins():
+    print('player1 you win \nplayer2 you lose')
+def player2wins():
+    print('player2 you win \nplayer1 you lose')    
+x = input('hello your are wanting to play a game of tic tac toe and make sure to have your own oppenent')
```
Comment (rakirs2) [inline DA/submissions/tictactoe.py final copy:5]: spelling of opponent at end

File: DA/submissions/tictactoe.py final copy [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    player12 = input(' player1/x pick between A1, A2, A3, B1, B2, B3, C1, C2, C3')
+    print(' a     b     c')  
+    print('1  -  |  -  |  - ') 
+    print(' _____|_____|_____')     
+    print('2  -  |  -  |  -')  
+    print(' _____|_____|_____')     
+    print('3  -  |  -  |  -')  
+    print('      |     |     ')
+    print(player12)
+    #asking for second input
```
Comment (rakirs2) [inline DA/submissions/tictactoe.py final copy:40]: asking who?

File: DA/submissions/tictactoe.py final copy [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        print('      |     |     ')
+         #asking for fifth input and tracking
+        player15=input('player2/o pick between A1, A2, A3, B1, B2, B3, C1, C2, C3')
+        print(player15)
+    if player1+player12+player13 or player12+player13+player14 or player1+player12+player14 or player1+player13+player14 or player1+player12+player15 or player1+player13+player15 or player12+player13+player15 or player1+player14+player15 or player12+player14+player15 or player13+player14+player15==('A1A2A3') or ('A1B1C1') or ('A1B2C3') or ('B1B2B3') or ('C1C2C3') or ('C1B2A3'):
+        player1wins()
+        'break'
+    else:
+        #nobody won so printing tie
+        print('tie')
```
Comment (rakirs2) [inline DA/submissions/tictactoe.py final copy:124]: So we have an algorithm that works but a lot of repeated code. Is there a mechanism we can use to reduce the code?

File: DA/submissions/tictactoe.py final copy [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        print(' _____|_____|_____')     
+        print('2  -  |  -  |  -')  
+        print(' _____|_____|_____')     
+        print('3  -  |  -  |  -')  
+        print('      |     |     ')
+        #asking for fourth input and tracking
+        player14=input('player1/x pick between A1, A2, A3, B1, B2, B3, C1, C2, C3')
+        
+        print(player14)
+    if player1+player12+player13 or player12+player13+player14 or player1+player12+player14 or player1+player13+player14==('A1A2A3') or ('A1B1C1') or ('A1B2C3') or ('B1B2B3') or ('C1C2C3') or ('C1B2A3'):
```
Comment (rakirs2) [inline DA/submissions/tictactoe.py final copy:90]: So are we actually doing what we think we're doing here?

**Review by rakirs2 — CHANGES_REQUESTED** [review]

File: DA/submissions/tictactoe.py final copy (hunk 3 above)
Comment (dandre2) [reply DA/submissions/tictactoe.py final copy:124]: i shortened it

File: DA/submissions/tictactoe.py final copy (hunk 2 above)
Comment (dandre2) [reply DA/submissions/tictactoe.py final copy:40]: Fixed it

File: DA/submissions/tictactoe.py final copy (hunk 1 above)
Comment (dandre2) [reply DA/submissions/tictactoe.py final copy:5]: fixed it

File: DA/submissions/tictactoe.py final copy (hunk 4 above)
Comment (dandre2) [reply DA/submissions/tictactoe.py final copy:90]: yes I think so

**Pull request closed without merging**